src/CMakeLists.txt

```diff
@@ -97,6 +97,7 @@ message(" ")
 message( STATUS "Configuring Level 5 packages --" )
 add_dir_if_exists( mesh )       # needs c4, ds++, mesh_element
 add_dir_if_exists( quadrature ) # needs mesh_element, parser, special_functions
+add_dir_if_exists( compton2 )     # needs ds++, units, cdi_analtyic
```

Comment:
I like the idea of renaming both, though my initial goal was to not break anything that depended on `compton`. If I rename the directory, will CMakeLists.txt in CTA/TRT/Jayenne be smart enough to know where to look?

Reply:
Thoughts on what to name the directories? I like the idea of having them separate, because they do very different things. `compton` wraps an interface to `csk` while `compton2` handles reading / interpolating csk data files itself.

`compton` -> `compton_wrapper`
`compton2` -> `compton_tools`

?

Reply:
How about csk_interface or compton_interface for the "wrapper" part, compton_data for the data "reading/handling" part?

Reply:
> I like the idea of renaming both, though my initial goal was to not break anything that depended on `compton`. If I rename the directory, will CMakeLists.txt in CTA/TRT/Jayenne be smart enough to know where to look?

Draco's build system publishes all of the path and library information automatically.  In cta, you will need to tell the code to link the libraries `compton_interface` and `csk_interface` (or whatever names we end up with).  Any C++ source files that currently has `#include compton/Compton.hh` will need to be updated.  This can be done easily with a sed command or through an IDE _refactor_ command.

Reply:
Alright, time for a vote: 

`compton` -> `compton_interface`

or

`compton` -> `compton_wrapper`

@warsa or @keadyk or @KineticTheory ?

I'm planning on using `compton2` -> `compton_tools` unless there is an objection

Reply:
I'm planning on using `compton_interface` unless there is a major objection

Reply:
Sounds good to me! :)

Reply:
I prefer it as well.